api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/aup/AcceptableUsagePolicyCoreProperties.java

```diff
@@ -50,4 +50,10 @@
      */
     private String aupPolicyTermsAttributeName;
 
+    /**
+     * AUP behaviour if status attribute is missing.
```

Comment:
You should refer to the "status" attribute via javadoc, and remove and explain exactly what this field does.

Reply:
I have reworded the description. I hope it is understood better.